test/protobuf_converter_test.dart

```diff
@@ -183,6 +186,189 @@ void main() {
             result.iif(success: id, failure: (_) => throw Exception()), value);
       });
     });
+
+    group("ConnectionStateUpdate conversion", () {
+      const id = 'id';
+      const connectionState = 1;
+      final message = pb.DeviceInfo()
+        ..id = id
+        ..connectionState = connectionState;
```

Comment:
Instances of the classes generated by protobuf are mutable. Ideally, mutable objects should be instantiated in a set-up block to avoid mutual test interference (in case one or more of the tests modify such an object).